app/helpers/studies_helper.rb

```diff
@@ -113,6 +113,23 @@ def eligibility_display(gender)
     end
   end
 
+  def render_healthy_volunteers(study)
+    rendered = '<div class="healthy-message" data-toggle="popover" data-title="Healthy Volunteer" data-content="A person who does not have the condition or disease being studied." data-placement="top">'
+            
+    if study.healthy_volunteers == true
+      rendered = rendered + '<i class="fa fa-check-circle"></i>This study is also accepting healthy volunteers <i class="fa fa-question-circle"></i>'
+    else
+      rendered = rendered + '<i class="fa fa-exclamation-triangle"></i>This study is NOT accepting healthy volunteers <i class="fa fa-question-circle"></i>'
+    end
+    rendered = rendered + '</div>'
+
+    return rendered.html_safe
+  end
+
+  def render_age_display(study)
+     return (study.respond_to?(:min_age_unit) and study.respond_to?(:max_age_unit)) ? age_display_units(study.min_age_unit, study.max_age_unit) : age_display(study.min_age, study.max_age)
```

Comment:
I know this code base is already loaded with `and` and `or`, but these are not meant to be used this way; they're meant for control flow. If you use them in lieu of `&&` and `||` it will *usually* be fine, but it can cause really gnarly issues:

https://avdi.codes/using-and-and-or-in-ruby/

Reply:
updated accordingly.